### PR TITLE
Building a contact form

### PR DESCRIPTION
**Pull request by NATASHA-ct**
- [ ] Added contact form with name, email, and message input.
- [ ] Added submit button interactions with hover effects
- [ ] Made sure user cannot submit the form if one or more of the fields is empty CSS
- [ ] Made my form to collect data by using the Formspree service

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,223 +9,264 @@
...
+      <div class="social">
+        <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+
+        <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+
+        <a href="#"><i class="fa fa-hand-peace-o" aria-hidden="true"></i></a>
+
+        <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+
+        <a href="#"> <i class="fa fa-medium" aria-hidden="true"></i></a>
```
Comment (tufoinnkuo10) [inline index.html:265]: - [ ] kindly increase the size of your icons. follow the figma dimensions

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @NATASHA-ct 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Form submits successfully


Required Changes ♻️
Check the comments under the review

**Optional suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 2]
```diff
@@ -9,223 +9,263 @@
...
+
+        <input class="email" type="email" id="email" name="email" placeholder="jeremy@gmail.com" required/>
+
+        <textarea
+          class="message"
+          name="message"
+          id="message"
+          maxlength="500"
+          placeholder="Enter text here"
+          required></textarea>
```
Comment (youmari) [inline index.html:247]: - [ ] as the previous reviewer asked you should use the right text color for your input as in the Figma design 
image

File: index.html [hunk 3]
```diff
@@ -9,223 +9,263 @@
...
+
+        <button id="see-project" class="see-prj" type="submit">See Project</button>
```
Comment (youmari) [inline index.html:249]: - [ ] [OPTIONAL] THIS should be named submit not See project so it would be better to change it

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @NATASHA-ct , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error 
✔️ Great title and short summary 
✔️ Github flow 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @NATASHA-ct  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 
✔️ Excellent job implementing the Figma Design
### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**